lib/facter/apt_gpg.rb

```diff
@@ -0,0 +1,10 @@
+require 'facter'
+
+Facter.add("apt_gpgversion") do
+  confine :osfamily => 'Debian'
+  setcode do
+    if File.executable? "/usr/bin/gpg"
+      Facter::Util::Resolution.exec("/usr/bin/gpg --version | head -n 1  | awk '{print $NF}'")
+    end
+  end
+end
```

Comment:
if we aren't warning on gpgversion I don't know if we still need this fact :\ sorry for sending you down this rabbit hole :(